tests/common/devices/sonic_asic.py

```diff
@@ -544,12 +544,13 @@ def portchannel_on_asic(self, portchannel):
                 if portchannel == pc:
                     return True
         return False
-    
+
     def write_to_config_db(self, dst_path):
         cmd = 'sonic-cfggen {} -j {} --write-to-db'.format(self.cli_ns_option, dst_path)
         return self.shell(cmd)
 
-    def get_portchannels_and_members_in_ns(self, tbinfo):
+    def get_portchannel_and_members_in_ns(self, tbinfo):
```

Comment:
@mannytaheri - what is the intention behind changing this function name without the corresponding changes in https://github.com/sonic-net/sonic-mgmt/pull/6341? Is this a merge issue?

This is currently causing PR test failures